homeassistant/components/vallox/__init__.py

```diff
@@ -171,7 +171,7 @@ async def async_update_data() -> ValloxState:
     hass.data[DOMAIN] = {"client": client, "coordinator": coordinator, "name": name}
 
     async def _async_load_platform_delayed(*_: Any) -> None:
-        await coordinator.async_refresh()
+        await coordinator.async_config_entry_first_refresh()
```

Comment:
This won't work because `async_config_entry_first_refresh` need to be called from `async_setup_entry`

Reply:
If you convert this to a config entry, you can get rid of the `_async_load_platform_delayed` and do this.  If it fails due to busy startup, the config entry will simply retry to setup later